src/main/java/net/sf/mzmine/util/StringCrypter.java

```diff
@@ -0,0 +1,123 @@
+package net.sf.mzmine.util;
+
+
+import de.unijena.bioinf.babelms.utils.Base64;
```

Comment:
Could you switch this to use java.util.Base64 class instead the de.unijena class?

Reply:
Of course, this was by accident :-)